api/envoy/config/filter/http/rate_limit/v2/rate_limit.proto

```diff
@@ -55,7 +55,6 @@ message RateLimit {
   // Configuration for an external rate limit service provider. If not
   // specified, any calls to the rate limit service will immediately return
   // success.
-  // [#comment:TODO(ramaraochavali): Make this required as part of cleanup of deprecated ratelimit
-  // service config in bootstrap.]
-  envoy.config.ratelimit.v2.RateLimitServiceConfig rate_limit_service = 7;
+  envoy.config.ratelimit.v2.RateLimitServiceConfig rate_limit_service = 7
+      [(validate.rules).duration.required = true];
```

Comment:
I don't think this is "duration.required" but rather message required? Same elsewhere?

Reply:
Yeah. Good catch. copy, paste and forgot to change